doc-push is broken

```
+ make html
python source/scripts/build_activation_images.py
/var/lib/jenkins/workspace/pytorch/docs/source/_static/img/activation/
ELU(alpha=1.0)
Traceback (most recent call last):
  File "source/scripts/build_activation_images.py", line 69, in <module>
    pylab.clf()
  File "/opt/conda/lib/python3.6/site-packages/matplotlib/pyplot.py", line 677, in clf
    gcf().clf()
  File "/opt/conda/lib/python3.6/site-packages/matplotlib/pyplot.py", line 592, in gcf
    return figure()
  File "/opt/conda/lib/python3.6/site-packages/matplotlib/pyplot.py", line 539, in figure
    **kwargs)
  File "/opt/conda/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 171, in new_figure_manager
    return cls.new_figure_manager_given_figure(num, fig)
  File "/opt/conda/lib/python3.6/site-packages/matplotlib/backends/backend_tkagg.py", line 1049, in new_figure_manager_given_figure
    window = Tk.Tk(className="matplotlib")
  File "/opt/conda/lib/python3.6/tkinter/__init__.py", line 2017, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
make: *** [figures] Error 1
Makefile:16: recipe for target 'figures' failed
```

CC @apaszke who was asking me about doc generation yesterday. Full log https://ci.pytorch.org/jenkins/job/pytorch-builds/job/doc-push/3092//console

And CC @pmitros

--- a/docs/source/scripts/build_activation_images.py
+++ b/docs/source/scripts/build_activation_images.py
@@ -7,6 +7,10 @@ online tutorials.
 import os.path
 import torch.nn.modules.activation
 import torch.autograd
+import matplotlib
+
+matplotlib.use('Agg')
+
 import pylab
 
 

